src/Makefile.am

```diff
@@ -942,6 +941,7 @@ libbitcoinkernel_la_SOURCES = \
   util/syserror.cpp \
   util/system.cpp \
   util/thread.cpp \
+  util/threadinterrupt.cpp \
```

Comment:
Is it really required in `libbitcoinkernel`?

I've managed to `./configure --with-experimental-kernel-lib --enable-experimental-util-chainstate && make` flawlessly without this source file.

Reply:
Opened #26360.